src/lib/products.ts

```diff
@@ -44,4 +44,6 @@ const products: ProductGroup[] = productSwitcherSlugs.map(
   }
 )
 
-export { products, productSlugsToNames }
+const productSlugs = Object.keys(productSlugsToNames) as ProductSlug[]
+
+export { products, productSlugs, productSlugsToNames }
```

Comment:
Update will be coming from https://github.com/hashicorp/dev-portal/pull/307

Reply:
Just merged! 🥳